simapp/app.go

```diff
@@ -509,7 +511,9 @@ func NewSimApp(
 	// create IBC module from bottom to top of stack
 	var transferStack porttypes.IBCModule
 	transferStack = transfer.NewIBCModule(app.TransferKeeper)
+	ibcAppRouter.AddRoute(ibctransfertypes.ModuleName, transferStack)
 	transferStack = ibcfee.NewIBCMiddleware(transferStack, app.IBCFeeKeeper)
+	ibcAppRouter.AddRoute(ibctransfertypes.ModuleName, transferStack)
```

Comment:
I'm not sure this second `AddRoute` is necessary? (looks like it is in testing/simapp/app.go so fine to leave, but I feel like it should be possible to remove)

Reply:
I think you are right, removed and e2e tests are passing!